src/Frost.test.ts

```diff
@@ -0,0 +1,84 @@
+import { describe } from 'riteway'
+import { getOptions } from './Frost'
+import { Method, StringifySecure } from './utils/utils'
+
+describe('Frost getOptions()', async (should: any) => {
+  const { assert } = should('')
+
+  for (const method of Object.keys(Method)) {
+    const currentMethod = (Method as any)[method]
+    {
+      const actual = getOptions(currentMethod)
+      const expected = {
+        method: currentMethod,
+        headers: new Headers({
```

Comment:
Can we change this (here an all other uses) to a plain object? The use of new Headers() implies knowledge of the implementation inside the function being tested.

Reply:
but it does not the same if I change it. if is a plan object I have to map the actual response also and it does not ensure you test. For another hand `new Headers` is the API of fetch

Reply:
do you have another option? @warrenv